app/models/subject_workflow_status.rb

```diff
@@ -36,7 +36,7 @@ def retire?
     !retired? && workflow.retirement_scheme.retire?(self)
   end
 
-  def retire!(reason=nil)
+  def retire!(reason=:classification_count)
```

Comment:
I'm wondering if we should do this higher-up (in `ClassificationLifecycle`) so that if some API user comes and retires without giving a reason, we don't assume it's due to classification count in that case.